.github/workflows/ci-p4tools.yml

```diff
@@ -39,4 +39,4 @@ jobs:
       # this is needed to create network namespaces for the ebpf tests.
     - name: Run tests (Ubuntu Linux)
       run: |
-        sudo docker run --privileged -w /p4c/build -e $CTEST_PARALLEL_LEVEL p4c ctest -R testgen-p4c-bmv2 -j2 --output-on-failure --schedule-random
+        sudo docker run --privileged -w /p4c/build -e $CTEST_PARALLEL_LEVEL p4c ctest -R testgen- -j2 --output-on-failure --schedule-random
```

Comment:
is this intentional?

Reply:
Yes, this will also include tests such as `testgen-clang-format` or `testgen-cpplint`. Next step is to merge those tests with top-level p4c.